app/views/content_items/consultation.html.erb

```diff
@@ -1,175 +1,193 @@
-  <% elsif @content_item.unopened? %>
-    This consultation opens <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %><br />
-    <span class="consultation-date"><time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time></span><br />
-    It closes at<br />
-    <span class="consultation-date"><time datetime="<%= @content_item.closing_date_time %>"><%= @content_item.closing_date %></time></span>
-  <% end %>
-<% end %>
 
-<% consultation_desc = capture do %>
-  <%= @content_item.description %>
-  <% if @content_item.held_on_another_website? %>
-    <br/><br/>
-    <strong>This consultation <% if @content_item.closed? %>was<% else %>is being<% end %> held on <a href="<%= @content_item.held_on_another_website_url %>">another website</a>.</strong>
-  <% end %>
-<% end %>
-<%= render 'components/banner', text: consultation_desc, title: 'Summary', aside: consultation_date %>
+    <% if @content_item.public_feedback_detail %>
+      <%= render 'components/heading', text: "Detail of feedback received", id: "public-feedback-detail-title" %>
+      <div aria-labelledby="public-feedback-detail-title">
+        <%= render 'govuk_component/govspeak',
+            content: @content_item.public_feedback_detail,
+            direction: page_text_direction %>
+      </div>
+    <% end %>
+
+    <% if @content_item.final_outcome? %>
+      <section class="original-consultation">
+        <header>
+          <%= render 'components/heading', text: "Original consultation", id: "original-consultation-title", heading_level: 2 %>
+        </header>
+    <% end %>
+
+    <% consultation_date = capture do %>
+      <% if @content_item.closed? %>
+        This consultation ran from<br /><span class="consultation-date"><time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time> to
+        <time datetime="<%= @content_item.closing_date_time %>"><%= @content_item.closing_date %></time></span>
+      <% elsif @content_item.open? %>
+        This consultation closes at<br />
+        <span class="consultation-date"><time datetime="<%= @content_item.closing_date_time %>"><%= @content_item.closing_date %></time></span>
+      <% elsif @content_item.unopened? %>
+        This consultation opens <% if @content_item.opening_date_midnight? %>on<% else %>at<% end %><br />
+        <span class="consultation-date"><time datetime="<%= @content_item.opening_date_time %>"><%= @content_item.opening_date %></time></span><br />
+        It closes at<br />
+        <span class="consultation-date"><time datetime="<%= @content_item.closing_date_time %>"><%= @content_item.closing_date %></time></span>
+      <% end %>
+    <% end %>
 
-<% if @content_item.final_outcome? %>
-  </section>
-<% end %>
+      <% consultation_desc = capture do %>
+        <%= @content_item.description %>
+        <% if @content_item.held_on_another_website? %>
+          <br/><br/>
```

Comment:
This looks a bit hacky - could it be a separate paragraph instead? ie wrap each thing in p tags ? Or is the spacing specifically trying to look different from a paragraph?

Reply:
This isn't really a change, it's a whitespace edit. If you review with `?w=1` in the Github URL you'll see these hacks were there originally. I think it's ok to leave as-is for now.